adhocracy4/images/validators.py

```diff
@@ -74,3 +76,21 @@ def validate_image(
     if errors:
         raise ValidationError(errors)
     return image
+
+
+@deconstructible
+class ImageAltTextValidator:
+    """Validate that if the input contains html img tags that all have the alt
+    attribute set, otherwise raise ValidationError.
+    """
+
+    message = _("Please add an alternative text for all images.")
+
+    def __call__(self, value):
+        soup = BeautifulSoup(value, "html.parser")
+        images = soup("img", alt=False)
+        if len(images) > 0:
```

Comment:
what is this checking? not obvious, mb add a comment?

Reply:
added a comment